blog/02-12-2023-Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT/Como-usar-a-metodologia-de-gestão-de-produtos-para-criar-soluções-com-NFT.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Como usar a metodologia de gestão de produtos para criar soluções com NFT
+description: Aprenda como usar a metodologia de gestão de produtos para criar soluções com NFT
+authors:
+- name: Evelyn Cordeiro
+  title: Founder of Inspiravalley
+  url: https://github.com/evelyncordeiro
+  image_url: https://avatars.githubusercontent.com/u/108406605
+tags: [beginner, celosage, erc721, smartcontract]
+hide_table_of_contents: false
```

Comment:
This has to be set it to true

Reply:
I looked at the others and are False. In the video tutorial that Viral explains how to make deploy, he speaks to leave as False. Let me know if I really need to change it.